pxd.c

```diff
@@ -2145,9 +2145,7 @@ static void pxd_abort_context(struct work_struct *work)
 	/* Let other threads see the value of allow_disconnected. */
 	synchronize_rcu();
 
-	spin_lock(&fc->lock);
```

Comment:
request_end(fc, req, false);
needs to be changed to 
request_end(fc, req, true); 
?

Reply:
Also how is list_empty check protected here?